src/shared_modules/content_manager/src/components/CtiApiDownloader.hpp

```diff
@@ -201,14 +201,14 @@ class CtiApiDownloader final : public AbstractHandler<std::shared_ptr<UpdaterCon
         {
             try
             {
-                m_urlRequest.get(HttpURL(m_url + queryParameters), onSuccess, onError, outputFilepath);
+                m_urlRequest.get(HttpURL(fullUrl), onSuccess, onError, outputFilepath);
                 retry = false;
             }
             catch (const cti_server_error& e)
             {
                 constexpr auto SLEEP_TIME_THRESHOLD {30};
 
-                logError(WM_CONTENTUPDATER, e.what());
+                logWarn(WM_CONTENTUPDATER, e.what());
```

Comment:
I think that we could have a message displaying the current value of `retryAttempt`, maybe as `logDebug2` 🤔

Reply:
Not sure about this. `retryAttempt` will stop incrementing itself when the `sleepTime` goes higher by 30 seconds. This would end up in something like the log below:
```
1 - Download failed: Some error.
2 - Download failed: Some error.
3 - Download failed: Some error.
3 - Download failed: Some error.
3 - Download failed: Some error.
3 - Download failed: Some error.
...
```